Cargo.toml

```diff
@@ -23,6 +23,7 @@ e2e-tests = []
 # enables dynamic cluster adjustments basing on status
 # changes server events
 unstable-dynamic-cluster = []
+cdrs_orm = ["serde"]
```

Comment:
`cdrs_orm` suggest what the feature will be used for, instead of what it does. `serde` or similar seems more appropriate.